components/ActionsList/index.js

```diff
@@ -20,11 +20,11 @@ const ActionsList = ({ actions }) => (
       </TableRow>
     </TableHead>
     <TableBody>
-      {actions.map(action => (
-        <TableRow key={action.title}>
+      {actions.map((action, i) => (
+        <TableRow key={`${action.summary}_${i}`}>
```

Comment:
Is the summary not unique already?

Reply:
remember we had this a few months ago when we we were filtering the customers activity feed in the SV? (and so the indexes were changing) :)

Reply:
hm, I think it wasn't unique for me because I was adding a lot of test actions, but if we're sure it won't have same titles, I'm happy to change it